genie-web/src/main/java/com/netflix/genie/web/spring/autoconfigure/health/HealthAutoConfiguration.java

```diff
@@ -49,22 +46,6 @@
 )
 public class HealthAutoConfiguration {
 
-    /**
-     * Provide a health indicator related to job memory usage if one hasn't already been provided elsewhere.
-     *
-     * @param jobMetricsService Implementation of {@link JobMetricsService} to use
-     * @param jobsProperties    The {@link JobsProperties} to use
-     * @return An instance of {@link GenieMemoryHealthIndicator}
-     */
-    @Bean
-    @ConditionalOnMissingBean(GenieMemoryHealthIndicator.class)
-    public GenieMemoryHealthIndicator genieMemoryHealthIndicator(
```

Comment:
This indicator is listed in the property `genie.tasks.cluster-checker.healthIndicatorsToIgnore` value.
Doesn't hurt to leave it, but should be removed.

Reply:
Done